### ISSUE TITLE
Crash report 8.9: NPE in ThemeStore.handleThemeActivated

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#6976]
I guess this needs to be addressed in FluxC, but logging the issue here for version tracking reasons.

```
Fatal Exception: org.greenrobot.eventbus.EventBusException: Invoking subscriber failed
       at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:505)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:487)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:479)
       at org.greenrobot.eventbus.AsyncPoster.run(AsyncPoster.java:46)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
       at java.lang.Thread.run(Thread.java:762)
Caused by java.lang.NullPointerException: Attempt to invoke virtual method 'void org.wordpress.android.fluxc.model.ThemeModel.setIsWpComTheme(boolean)' on a null object reference
       at org.wordpress.android.fluxc.store.ThemeStore.handleThemeActivated(ThemeStore.java:416)
       at org.wordpress.android.fluxc.store.ThemeStore.onAction(ThemeStore.java:237)
       at java.lang.reflect.Method.invoke(Method.java)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:485)
       at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:479)
       at org.greenrobot.eventbus.AsyncPoster.run(AsyncPoster.java:46)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
       at java.lang.Thread.run(Thread.java:762)
```
https://fabric.io/automattic/android/apps/org.wordpress.android/issues/5a148f6561b02d480df4446c?time=last-ninety-days

**Comment by oguzkocer** [comment]
I am working on a fix in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/645 but as mentioned in that PR, we should look into why the crash is happening which I have failed to do so, at least for now, and hoping @tonyr59h can help with that.

**Comment by oguzkocer** [comment]
I was able to find a way to reproduce this issue:

1. Activate a theme for a Jetpack site
2. Activate the same theme for a wp.com site

**Comment by oguzkocer** [comment]
Fixed in #6996.
